### DISCUSSION TITLE
Are you guys getting errors?

### DISCUSSION THREAD

**Discussion by Emend026** [anhourandaquarter/carmina#33] [Q&A]
<img width="1396" alt="Screen Shot 2024-12-09 at 5 39 52 PM" src="https://github.com/user-attachments/assets/492683e3-d43d-4194-ace2-336ff30946ba">

Are you guys getting these errors? Do you know why these are showing up?

**Comment by infinite0x20** [comment]
Yeah, that has to do with something in the environment yml. I think if you click on the "build-linux" link under Annotations it might show you more detailed information.

**Reply by Emend026** [reply]
I think it has to do with the 're' package being there. online says the package re doesn't need to be installed via conda or pip. including it in the yml will cause an error. Went on the yml branch to remove it but it's still giving me an error. any thoughts?
